pages/vi/treehouses-remote.md

```diff
@@ -10,6 +10,9 @@ On an Android device, find the [treehouses remote](https://play.google.com/store
 
 ![](images/remoteiconsml.jpg)
 
+_**Rate the app**_.
+Follow [this link](https://play.google.com/apps/testing/io.treehouses.remote) to join and download the beta version.
```

Comment:
hmm this is a not beta right ;)

we switch to beta in the orientation

Reply:
?
it leads me to beta:
![image](https://user-images.githubusercontent.com/50122974/66956769-fde22c80-f032-11e9-91f8-c18f4430a2c0.png)

Reply:
ya you flipped the switch already

Reply:
ooh ... I see now ... they should install official version first and then switch in a later (orientation) step to testing

Reply:
reverted back